docs/FSDP_example.py

```diff
@@ -0,0 +1,126 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
We do not recommend to put an example script in `docs` without a document. The example script can be an auxiliary of a document. For example, https://github.com/awslabs/ratex/blob/main/docs/QUICKSTART.md uses `lenet.py`.